proprietary/tokens/src/components/ams/ordered-list.tokens.json

```diff
@@ -3,12 +3,22 @@
     "ordered-list": {
       "color": { "value": "{ams.color.primary-black}" },
       "font-family": { "value": "{ams.text.font-family}" },
-      "font-size": { "value": "{ams.text.level.5.font-size}" },
       "font-weight": { "value": "{ams.text.font-weight.normal}" },
       "gap": { "value": "0.75rem" },
       "inverse-color": { "value": "{ams.color.primary-white}" },
-      "line-height": { "value": "{ams.text.level.5.line-height}" },
       "list-style-type": { "value": "decimal" },
+      "small": {
+        "font-size": { "value": "{ams.text.level.6.font-size}" },
+        "line-height": { "value": "{ams.text.level.6.line-height}" }
+      },
+      "medium": {
+        "font-size": { "value": "{ams.text.level.5.font-size}" },
+        "line-height": { "value": "{ams.text.level.5.line-height}" }
+      },
```

Comment:
Please move these back to the root level. These sizes belong to the default appearance of the element, not to a variation – and `medium` is not an explicit option for the `size` prop. This is consistent with `UnorderedList` and I will add a ticket to change this for `Paragraph` as well.